couchdb-cluster-admin/size-shards.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# usage:
+#   bash couchdb-cluster-admin/size-shards.sh <remote-ip> <remote-data-dir>
+
+ip=$1
+data_dir=$2
+ssh ${ip} "cd ${data_dir}/shards; find . -type f | xargs ls -alS | xargs -L 1 echo | cut -d' ' -f5,9 | sed 's|\./||g'"
```

Comment:
Can you assume that all shards are the same size? (I think you can)

If so you can use http://docs.couchdb.org/en/2.0.0/api/database/common.html#get--db to get data/disk size and divide by number of shards. That way it doesn't require ssh commands and it fixes the case when you don't have one server that has all the shards

Reply:
That's a good point. The shards aren't _exactly_ the same size, but probably close enough to make that worth it. I'll do that in a follow up when I get back to that.